test/epoch_token_locker.js

```diff
@@ -35,30 +38,32 @@ contract("EpochTokenLocker", async (accounts) => {
       await epochTokenLocker.deposit(ERC20.address, 100)
 
       assert.equal(await epochTokenLocker.getPendingDepositAmount(user_1, ERC20.address), 200)
-      assert.equal(await epochTokenLocker.getPendingDepositBatchNumber(user_1, ERC20.address), 0)
     })
 
     it("does not consolidates two deposits, if they are not deposited during same stateIndex", async () => {
       const epochTokenLocker = await EpochTokenLockerTestInterface.new()
       const ERC20 = await MockContract.new()
       await ERC20.givenAnyReturnBool(true)
       await epochTokenLocker.deposit(ERC20.address, 100)
-      await epochTokenLocker.increaseStateIndex()
+      await waitForNSeconds(301)
```

Comment:
For this things is better to use constants that are defined in a single place in the project. 
This way, on top of being more easy to read `BATCH_TIME_SECONDS + 1`, we only have to change it on place if we go from 5 min batches to 3 min batches for example.

Reply:
Yes, we should probably do this across all unit tests.